nixos/modules/services/hardware/udev.nix

```diff
@@ -202,12 +202,22 @@ in
         '';
       };
 
-      extraRules = mkOption {
+      initrdRules = mkOption {
         default = "";
         example = ''
           SUBSYSTEM=="net", ACTION=="add", DRIVERS=="?*", ATTR{address}=="00:1D:60:B9:6D:4F", KERNEL=="eth*", NAME="my_fast_network_card"
         '';
         type = types.lines;
+        description = ''
+          <command>udev</command> rules to include in the initrd. They'll
+          be written into file <filename>99-local.rules</filename>. Thus
+          they are read and applied after all other rules.
+        '';
+      };
+
+      extraRules = mkOption {
+        default = "";
+        type = types.lines;
```

Comment:
We probably should still keep an example here. Maybe

```
ENV{ID_VENDOR_ID}=="046d", ENV{ID_MODEL_ID}=="0825", ENV{PULSE_IGNORE}="1"
```

?

Reply:
Good idea, I'll grab this example.

Reply:
Added.